### DISCUSSION TITLE
Unstaged changes not showing on new computer

### DISCUSSION THREAD

**Discussion by govi218** [magit/magit#4567] [Support] [answered]
Hi all, I've had no issues with magit until I loaded emacs on my new machine ([frame.work](https://frame.work) fwiw). All the functionality seems to be good, except the status buffer does not show unstaged changes at all.

In the first pic you can see the empty `Unstaged Changes` section of the status buffer. In the second pic you can see that my terminal shows unstaged changes. I don't see any logs in my error buffer but I could be looking in the wrong place. Any help is much appreciated!

![magit-1](https://user-images.githubusercontent.com/8207304/147546810-a726bd59-d749-4aa7-8ab0-20fafd0c47ee.png)
![magit-2](https://user-images.githubusercontent.com/8207304/147546813-65e4972a-2cd0-440d-ac89-b50227b0fdee.png)

**Comment by tarsius** [comment, marked as answer]
Some error occurred while magit was trying to insert the unstaged files. You should get a backtrace if you `M-x toggle-debug-on-error` before refreshing the buffer.

**Reply by govi218** [reply]
Did it and realized I was missing delta. Thanks!
